### ISSUE TITLE
need integration tests for Electron app

### ISSUE THREAD

**Issue by rbuels** [GMOD/jbrowse#983]
Using Spectron.

Should test all of the Electron specific things, particularly the welcome screen, saving sessions, loading sessions.

**Comment by cmdcolin** [comment]
Have an initial test of electron functionality that works on travis-CI here https://github.com/GMOD/jbrowse/compare/add_new_spectron_test

Note that electron dependencies and spectron are manually installed...figured they do not need to be included via package.json for majority of people

**Comment by rbuels** [comment]
@cmdcolin assigning this to 1.14.1.  Do you think you can have some fair spectron test coverage in time for that release?  probably at least a few days ahead of that milestone due date.

**Comment by cmdcolin** [comment]
Seems like it should be possible. I guess more tests can be obtained by taking the xpath queries from selenium tests and converting into the new test runner.

**Comment by rbuels** [comment]
Yeah, mostly just testing the functionality that is unique to the electron app.

**Comment by cmdcolin** [comment]
The test in #1028 currently does some baseline testing of electron which includes

a) app startup, check welcome screen
b) open sequence file
c) restart and check that it is in the previous sessions screen

I could maybe one for opening a track inside the browser and then saving session.
